src/lib/modules/console/suggestions.ts

```diff
@@ -75,7 +75,11 @@ export default class Suggestions {
     suggestions.push({value: "web3", command_type: "javascript object", description: "instantiated web3.js object configured to the current environment"});
     suggestions.push({value: "EmbarkJS", command_type: "javascript object", description: "EmbarkJS static functions for Storage, Messages, Names, etc."});
 
-    if (cmd.indexOf(".") > 0) {
+    if (cmd.indexOf(".") <= 0) {
+      return cb(fuzzySearch(cmd, suggestions, (suggestion: Suggestion) => suggestion.value + " " + suggestion.description).map((x: any) => x.original));
```

Comment:
Could you create a function out of the callback (that returns the suggestion). Would reduce duplication